assets/evaluation_results/gsm8k_gpt-4o_chat_completion/spec.yaml

```diff
@@ -0,0 +1,31 @@
+type: evaluationresult
+name: gsm8k_gpt-4o_chat_completion
+version: 2.13.06
+display_name: gsm8k_gpt-4o_chat_completion
+description: gsm8k_gpt-4o_chat_completion
+dataset_family: gsm8k
+dataset_name: gsm8k
+
+model_name: gpt-4o
+model_version: "5/13/2024"
```

Comment:
please verify the version format

Reply:
this is the same model version that gets displayed in the azure openai deployment for gpt-4o model: [Deployments | Azure OpenAI Studio](https://oai.azure.com/portal/604cd572fc474c2d8755156de28cf598/deployment/gpt-4o?tenantid=72f988bf-86f1-41af-91ab-2d7cd011db47)